tests/components/intellifire/test_config_flow.py

```diff
@@ -1,55 +1,207 @@
 """Test the IntelliFire config flow."""
-from unittest.mock import AsyncMock, MagicMock
+from unittest.mock import AsyncMock, MagicMock, patch
 
 from homeassistant import config_entries
+from homeassistant.components.intellifire.config_flow import MANUAL_ENTRY_STRING
 from homeassistant.components.intellifire.const import DOMAIN
+from homeassistant.const import CONF_HOST
 from homeassistant.core import HomeAssistant
 from homeassistant.data_entry_flow import RESULT_TYPE_CREATE_ENTRY, RESULT_TYPE_FORM
 
+from tests.common import MockConfigEntry
 
-async def test_form(
+
+async def test_no_discovery(
+    hass: HomeAssistant,
+    mock_setup_entry: AsyncMock,
+    mock_intellifire_config_flow: MagicMock,
+) -> None:
+    """Test we should get the manual discovery form - because no discovered fireplaces."""
+    with patch(
+        "intellifire4py.udp.AsyncUDPFireplaceFinder.search_fireplace",
```

Comment:
```suggestion
        "homeassistant.components.intellifire.config_flow.AsyncUDPFireplaceFinder.search_fireplace",
```
Please patch where its imported only